index.html

```diff
@@ -655,7 +657,41 @@ <h4>LET'S CONNECT</h4>
 
          </div>
 
-     </section>
+     </div>
+
+     <div class="contacto">
+
+         <div class="flex-column fondo">
+
+             <h2 class="margen">Contact me</h2>
+
+             <p class="margen">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it!</p>
+
+             <form action="https://formspree.io/f/mayzvpoz" method="post" class="margen">
+
+                 <input type="text" name="name" maxlength="30" required placeholder="Lorem Ipsum">
+
+                 <input type="email" required placeholder="loremipsum@gmail.com">
+
+                 <textarea maxlength="500" rows="4" cols="50" required placeholder="Write your message here..."></textarea>
+
+                 <div>
+                    
+                     <button id="boton" type="submit" >
+                        
+                         Get in touch
+
+                     </button>
+
+                 </div>
+
+             </form>
```

Comment:
- [ ] Kindly ensure that your Form is properly centered on the page as seen in the Figma Design :+1: 

| Your Design | The Figma Design |
|--------|--------|
| ![image](https://github.com/maximoortelli/Portfolio-setup-and-mobile--first/assets/102699958/a1b9905e-e2c6-464a-b886-1a1a67cfafdf) | ![image](https://github.com/maximoortelli/Portfolio-setup-and-mobile--first/assets/102699958/bb38d95d-ea26-45b9-b0b8-f8ce78fd17fe) |

Reply:
Done, thank you NahnahAJ 👍😃

Reply:
Done, thank you NahnahAJ 👍😃